packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -524,12 +527,11 @@ export class GarbageCollector implements IGarbageCollector {
          * 2. GC should not be disabled via disableGC GC option.
          * These conditions can be overridden via runGCKey feature flag.
          */
-        this.shouldRunGC = this.mc.config.getBoolean(runGCKey) ?? (
-            // GC must be enabled for the document.
-            this.gcEnabled
-            // GC must not be disabled via GC options.
-            && !this.gcOptions.disableGC
-        );
+        this.shouldRunGC = false; // Until TEMPORARY measure above hardcoding snapshotCacheExpiryMs is removed, disable
```

Comment:
Why is this false? We will literally turn off GC for everyone if this is false

Reply:
This includes mark phase

Reply:
OHH dang meant to tweak shouldRunSweep

Reply:
shouldRunSweep isn't even referenced. But just to be safe.